public/index.html

```diff
@@ -309,8 +309,8 @@ <h2 id="faq">FAQ</h2>
           </details>
 
           <details class="FAQItem" id="how-does-the-test-work">
-            <summary><span>How does the test work?</span></summary>
-            <p>In order to test if your ISP is implementing BGP safely, we announce a legitimate route but we make sure the announcement is <em>invalid</em>. If you can load the website we host on that route, that means the invalid route was accepted by your ISP. A leaked or a hijacked route would likely be accepted too.</p>
+            <summary><span>Does my ISP support RPKI?</span></summary>
```

Comment:
While I certainly agree with the principle that ideally the FAQ items would stand well on their own (in a vacuum, as you said), I also see a large value in having the phrase "How does the test work?" called out for folks who did run the test and then continued to scroll the page.

Rather than removing the title “How does the test work?” I wonder if there’s a way we could re-word it to keep the "test" in there but still improve it’s ability to be understood without context.